pages/api/auth/[...nextauth].ts

```diff
@@ -88,9 +88,9 @@ export const authOptions = {
       if (session?.user) {
         session.user._id = token.uid;
         session.user.status = token.status;
+        session.user.name = token.name;
       }
```

Comment:
Use session to store user data. Can be handy whenever you need frequent access to user data.